.gitmodules

```diff
@@ -7,3 +7,7 @@
 	url = https://github.com/tikv/titan.git
 	branch = master
 
+[submodule "librocksdb_sys/rocksdb-cloud"]
+	path = librocksdb_sys/rocksdb-cloud
```

Comment:
I would like to have it not under `librocksdb_sys`. Please place the submodule under `librocksdb_cloud_sys/rocksdb-cloud`.